.env.local

```diff
@@ -7,3 +7,9 @@ CLUSTERED_MODE=false
 DEBUG=0
 BASHLOG_FILE=0
 BASHLOG_FILE_PATH=platform.log
+
+# Disables Fhir validation agains IG
+DISABLE_VALIDATION=true
+
+# Name of folder containing the Fhir IG importer scripts
+FHIR_CONFIG_IMPORTER_FOLDER_NAME=hapi-fhir
```

Comment:
Same comment with Michael about defining the env vars only for override

Reply:
addressed